### DISCUSSION TITLE
access is denied for nvm use 18.3.0

### DISCUSSION THREAD

**Discussion by rootsmusic** [coreybutler/nvm-windows#818] [Support] [answered]
Where can I find _novice_ instructions to install node and npm?  I don't understand why I'm receiving the following error.  I don't know Bash, nvm, node, npm and Git.  Thanks.

**Steps:**
1. Installed on Windows 10 using nvm-setup.exe.
2. Open Git Bash 2.36.1.windows.1.
3. `nvm -v` shows "running version 1.1.9".
4. `nvm install latest` confirms "installation complete".
5. `nvm use 18.3.0` says: "exit status 1: Access is denied."

**Comment by rootsmusic** [comment, marked as answer]
Git Bash needs to "run as administrator".  (I didn't think about it since Git Bash successfully installed the latest version of nvm.)

**Comment by piyush26arora** [comment]
Can I install this on top of windows 7 ?
I'm having a hard time getting a proper Node version installed on my laptop to be able to run React.js .(Node 14>=)
Last supported version for Windows 7 was 13.14._
![image](https://user-images.githubusercontent.com/83628525/174024904-7823a264-a75e-4c82-830c-fcf1c7e50132.png)
Kindly help.

**Reply by coreybutler** [reply]
Please see [Support-for-Windows-7-and-Earlier](https://github.com/coreybutler/nvm-windows/wiki/Support-for-Windows-7-and-Earlier).
